rest_auth/registration/views.py

```diff
@@ -91,31 +95,96 @@ def post(self, request, *args, **kwargs):
         return Response({'detail': _('ok')}, status=status.HTTP_200_OK)
 
 
-class SocialLoginView(LoginView):
```

Comment:
I moved the `SocialLoginView` definition into the conditional `if` block where the imports reside, I think this cleans up the code slightly.